packages/kotti-ui/source/props.ts

```diff
@@ -0,0 +1,54 @@
+import { cloneDeep } from 'lodash'
+import { z } from 'zod'
+
+/**
+ * Easily validate vue props, improve error messages drastically
+ *
+ * This augments/replaces Vue’s prop.validator approach by using zod and therefore actually
+ * prints errors to the console that can actually help developers rather than just say
+ * "hey you failed the validator, good luck!" as vue would do by default.
```

Comment:
rather than the non-distinct `vue-warn` message about a validator failing for a component.